container_files/tutorials/Selecting Columns Programmatically Via a Column Expression Tutorial.ipynb

```diff
@@ -0,0 +1,1208 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Selecting Columns Programmatically Via a Column Expression Tutorial\n",
+    "\n",
+    "MLDB provides a complete implementation of the [SQL](../../../../doc/#builtin/sql/Sql.md.html) SELECT statement. Most of the functions you are accustomed to using [are available](../../../../doc/#builtin/sql/ValueExpression.md.html) in your queries. \n",
+    "\n",
+    "MLDB is different from traditional SQL databases in that there is no enforced schema on rows, allowing you to work with millions of columns of sparse data. This makes it easy to load and manipulate sparse datasets. To reduce the size of your dataset or use only specific variables, we may need to select columns based on specific critera. [Column Expressions](../../../../doc/#builtin/sql/SelectExpression.md.html) is an MLDB extension that provides additional control over your column selection. A column expression effectively creates a table with columns selected via a SELECT statement.\n",
```

Comment:
"A column expression effectively creates a table with columns selected via a SELECT statement." -> pe genre allows a user to programatically pick which columns to return in the select